### DISCUSSION TITLE
Multiselect constraints

### DISCUSSION THREAD

**Discussion by FlaminWrap** [Budibase/budibase#9386] [Help] [answered]
Hello,

I was wondering if there was a way to limit the amount of options selected in a Multiselect field.

I have six options available but I only want a maximum of three to be selected or else the form fails validation.

Thanks

**Comment by andz-bb** [comment, marked as answer]
Hey @FlaminWrap 

I couldn't find a way to achieve this with native form validation, although I might be missing something there as well. I did find a workaround though using On Change actions. Seems to work well!

![multiselect limit](https://user-images.githubusercontent.com/110921612/213484614-686f2c64-47c8-44b7-8030-a9555fdc15df.gif)

Here's how I set up my Multi-select picker component's actions:

<img width="1421" alt="image" src="https://user-images.githubusercontent.com/110921612/213487785-5a50bc0e-2eda-405f-b080-52f31f0f8c76.png">

Hope this helps!

**Reply by melohagan** [reply]
Clever solution and great answer!

**Comment by FlaminWrap** [comment]
Thank you, that works perfectly!
